docs/RFCS/explain_debug.md

```diff
@@ -0,0 +1,71 @@
+- Feature Name: explain
+- Status: draft
+- Start Date: 2015-08-10
+- RFC PR:
+- Cockroach Issue: #2047
+
+# Summary
+
+Add support for an `EXPLAIN DEBUG` statement. The `EXPLAIN DEBUG` statement
+will cause simple (single table or index) select statements to display the
+underlying key-value pairs that were accessed:
+
+* the first column will contain the index of the row the key maps to.
+* the second column contain will contain a pretty-printed version of the key in
+  the underlying key-value store.
+* the third column will contain a pretty-printed version of the value.
+* the fourth column will contain a boolean indicating whether the row was
+  filtered by the where-clause or not.
```

Comment:
would you mind having a three-line example here? I'm not 100% on what the first row would look like.